app/models/pipeline_run.rb

```diff
@@ -139,4 +150,17 @@ def download_file(s3_path, destination_dir)
     return nil unless status.exitstatus.zero?
     "#{destination_dir}/#{File.basename(s3_path)}"
   end
+
+  def file_generated_since_run(s3_path)
+    command = "aws s3 ls #{s3_path}"
+    stdout, _stderr, status = Open3.capture3(command)
+    return false unless status.exitstatus.zero?
+    s3_file_time = DateTime.strptime(stdout[0..18], "%Y-%m-%d %H:%M:%S")
+    (s3_file_time > created_at)
```

Comment:
Are we guaranteeing that the timezone in S3 and in the app are the same?

Reply:
They are all UTC